README.rst

```diff
@@ -73,27 +80,28 @@ one. For example::
             [ 2 ]: S3Admin
     Selection: 2
 
-To switch roles, first log out, then log in again selecting a different
-role. Note that if you log in to the same IdP using the same username,
-you will not be prompted for your password or Duo factor until
-the IdP session expires::
+To switch roles, first log out, then log in again selecting a
+different role. Note that if you log in to the same IdP using the
+same username, you will not be prompted for your password or Duo
+factor until the IdP session expires::
 
     $ aws logout
     $ aws login
-    Username [netid]: 
+    Username [netid]:
```

Comment:
@ddriddle, you state that the user won't be prompted again if their IdP session hasn't expired, but then your example shows them logging in again. Yes, this implies that their IdP session has expired, but it's slightly confusing. Suggest removing the username, password, and factor prompts from the logout and log back in example.

Reply:
Never mind, I missed that there was only a username prompt, no prompts for password or second factor. This is fine.